### ISSUE TITLE
init no longer works.

### ISSUE THREAD

**Issue by antony** [QingWei-Li/docsify-cli#9]
```
$ docsify init .
docsify init <path> [--local] [--theme vue]

Options:
  --help, -h     Show help                             [boolean] [default: true]
  --version, -v  Show version number                                   [boolean]
  --local, -l    Copy docsify files to local.         [boolean] [default: false]
  --theme, -t    Theme file to be used.
             [string] [choices: "vue", "buble", "dark", "pure"] [default: "vue"]
```

It seems like init is expecting an optional argument or some such - when you run it, it simply prints the help text and doesn't do anything.

**Comment by jrappen** [comment]
@antony This is caused by QingWei-Li/docsify-cli@54567bc, which added `--help` as a default parameter added to **every** command. Hence running `docsify init (....whatever...)` would have the option `--help` added, which therefore brings up the help menu instead of running the command. A quick fix until this fixed here would be to delete this line:

<https://github.com/QingWei-Li/docsify-cli/blob/54567bc/bin/docsify#L75>

**Comment by QingWei-Li** [comment]
Fixed in `v3.2.0`
